data/mi/committees/upper-Appropriations-5d0b75a6-8447-4948-9d31-d8ea55ae6a02.yml

```diff
@@ -1,20 +1,20 @@
 id: ocd-organization/5d0b75a6-8447-4948-9d31-d8ea55ae6a02
 jurisdiction: ocd-jurisdiction/country:us/state:mi/government
 classification: committee
-name: Appropriations
+name: Michigan Senate Appropriations Committee
```

Comment:
This is a bug. We shouldn't merge changes like this until it is fixed.

Reply:
Right, it's happening across jurisdictions specifically for senate. Fixed some manually before merging but should fix the script

Reply:
Thank you! If you can fix easily that is good. But I will look into it soon.